src/EnergyPlus/Furnaces.cc

```diff
@@ -866,6 +885,8 @@ namespace Furnaces {
 		int SteamIndex; // steam coil index
 		Real64 SteamDensity; // density of steam at 100C
 		int DXCoilIndex; // Index to DX coil in HXAssited object
+		std::string IHPCoilName;//IHP cooling coil name
+		int IHPCoilIndex(0);//IHP cooling coil id
```

Comment:
These new locals are confusing since they refer to different coils at different places in GetInput. Why not just use existing locals CoolingCoilName and HeatingCoilName? or if a new coil type is needed then be explicit like using CondenserCoilName.